src/App.js

```diff
@@ -26,6 +41,9 @@ class Container extends React.Component {
   }
 
   playSound() {
```

Comment:
This method should reset the progress to 0. If you click double click the soundbutton, the progress bar will infinitely loop. The progress bar should start over on each click.

Reply:
Fixed